javascript/node/processes-clustering/manage-child-processes-in-node.md

```diff
@@ -1,22 +1,10 @@
 ---
 author: catalin
 
-levels:
-  - basic
-  - advanced
-  - medium
-
 type: normal
 
 category: how to
 
-standards:
-  javascript.node-standard-library-os.5: 20
-
-tags:
-  - workout
-  - deep
-
 links:
   - '[www.graemeboy.com](http://www.graemeboy.com/node-child-processes){website}'
   - '[krasimirtsonev.com](http://krasimirtsonev.com/blog/article/Nodejs-managing-child-processes-starting-stopping-exec-spawn){website}'
```

Comment:
`graemeboy` domain is up for sale now, so i have removed the resource. also updated description of the other resource

```suggestion
links:
  - '[Managing Child Processes](http://krasimirtsonev.com/blog/article/Nodejs-managing-child-processes-starting-stopping-exec-spawn){website}'
```